src/Renderer.h

```diff
@@ -96,6 +96,7 @@ void RenderFullscreenQuad();
 bool ReloadShader( const char * szShaderCode, int nShaderCodeSize, char * szErrorBuffer, int nErrorBufferSize );
 void SetShaderConstant( const char * szConstName, float x );
 void SetShaderConstant( const char * szConstName, float x, float y );
+void SetShaderConstant( const char * szConstName, unsigned int num, float* data );
```

Comment:
(oops, this function was added for a different change. will come with another PR)